Products/Products-Root/Screens/Authentication/Login/LoginViewController.swift

```diff
@@ -39,6 +42,21 @@ class LoginViewController: BaseViewController, AlertPresenter, LoadingViewPresen
     @IBAction func loginButtonPressed(_ sender: Any) {
         viewModel.login()
     }
+    
+    @IBAction func forgotPasswordButtonPressed(_ sender: Any) {
+    }
+    
+    @IBAction func showPasswordButtonPressed(_ sender: Any) {
+        if (passwordHidden == true) {
```

Comment:
no need to add parentheses. Please install swiftlint, it would have raised a warning here